testv2/e2e/scenario_upgrade.go

```diff
@@ -43,13 +45,16 @@ func (scenario *scenarioUpgrade) SetVersions(versions ...string) {
 }
 
 func (scenario *scenarioUpgrade) Run(t *testing.T) {
-	t.Helper()
+	if err := makeBin("build").Run(); err != nil {
+		t.Fatalf("building kubeone: %v", err)
```

Comment:
> building kubeone: %v

Failure messages usually have a negative connotation, this should rather be `failed building kubeone binary: %v` or something similar.

Reply:
If you agree with that then most of the `t.Fatalf` blocks need ammendments.

Reply:
No, I don't agree, recently I've rebuild whole errors handling in kubeone and messaging to this form `"doing something: %v", err` which turns into `"doing something: process exited with error 1"` or similar.